Formula/o/openapi-tui.rb

```diff
@@ -0,0 +1,38 @@
+class OpenapiTui < Formula
+  desc "TUI to list, browse and run APIs defined with openapi spec"
+  homepage "https://github.com/zaghaghi/openapi-tui"
+  url "https://github.com/zaghaghi/openapi-tui/archive/refs/tags/0.10.0.tar.gz"
+  sha256 "59ab143671843c5dc16056900b3c2413cc58a943f545ea2d94d687568410cb30"
+  license "MIT"
+  head "https://github.com/zaghaghi/openapi-tui.git", branch: "main"
+
+  depends_on "pkgconf" => :build
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "openssl@3"
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/openapi-tui --version")
+
+    # Fails in Linux CI with `No such device or address (os error 6)`
+    return if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]
+
+    openapi_url = "https://raw.githubusercontent.com/Tufin/oasdiff/8fdb99634d0f7f827810ee1ba7b23aa4ada8b124/data/openapi-test1.yaml"
+
+    begin
+      output_log = testpath/"output.log"
+      pid = spawn bin/"openapi-tui", "--input", openapi_url, [:out, :err] => output_log.to_s
+      sleep 1
+      assert_match "APIs", output_log.read
+    ensure
+      Process.kill("TERM", pid)
+      Process.wait(pid)
+    end
+  end
```

Comment:
There is no process after this, so we may use the `ensure` for the whole test not only `begin`.
```suggestion
    output_log = testpath/"output.log"
    pid = spawn bin/"openapi-tui", "--input", openapi_url, [:out, :err] => output_log.to_s
    sleep 1
    assert_match "APIs", output_log.read
  ensure
    Process.kill("TERM", pid)
    Process.wait(pid)
  end
```

Reply:
works for me 👍